delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/SupervisionRoutes.scala

```diff
@@ -13,34 +15,44 @@ import ch.epfl.bluebrain.nexus.delta.sdk.directives.DeltaDirectives.emit
 import ch.epfl.bluebrain.nexus.delta.sdk.directives.UriDirectives.baseUriPrefix
 import ch.epfl.bluebrain.nexus.delta.sdk.directives._
 import ch.epfl.bluebrain.nexus.delta.sdk.identities.Identities
+import ch.epfl.bluebrain.nexus.delta.sdk.marshalling.RdfMarshalling
 import ch.epfl.bluebrain.nexus.delta.sdk.model.BaseUri
 import ch.epfl.bluebrain.nexus.delta.sdk.permissions.Permissions.supervision
-import ch.epfl.bluebrain.nexus.delta.sourcing.stream.{SupervisedDescription, Supervisor}
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.ProjectRef
+import ch.epfl.bluebrain.nexus.delta.sourcing.stream.SupervisedDescription
 import io.circe.Encoder
 import io.circe.generic.semiauto.deriveEncoder
-import kamon.instrumentation.akka.http.TracingDirectives.operationName
+import io.circe.syntax.EncoderOps
 
 class SupervisionRoutes(
     identities: Identities,
     aclCheck: AclCheck,
-    supervised: IO[List[SupervisedDescription]]
+    supervised: IO[List[SupervisedDescription]],
+    projectsHealth: IO[List[ProjectRef]]
```

Comment:
Could maybe pull out little traits for these, like the health operation of `Projects` - as a way to inject it granularly but a little easier to navigate. E.g. `FetchFileResource` and `FetchFileStorage` for files. It's less necessary here though since there's at least a giga-trait in the first place

Reply:
I will add this